src/applications/login/containers/SignInApp.jsx

```diff
@@ -0,0 +1,233 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+import AlertBox from '@department-of-veterans-affairs/formation-react/AlertBox';
+
+import ExternalServicesError from 'platform/monitoring/external-services/ExternalServicesError';
+// import recordEvent from 'platform/monitoring/record-event';
+import SubmitSignInForm from 'platform/static-data/SubmitSignInForm';
+import { login, signup } from 'platform/user/authentication/utilities';
+import environment from 'platform/utilities/environment';
+
+import downtimeBanners from '../utilities/downtimeBanners';
+
+const loginHandler = loginType => () => {
+  // TODO add separate login tracking for this page
+  //   recordEvent({ event: `login-attempted-${loginType}` });
+  login(loginType, 'v1');
+};
+
+const handleDsLogon = loginHandler('dslogon');
+const handleMhv = loginHandler('mhv');
+const handleIdMe = loginHandler('idme');
+
+const vaGovFullDomain = environment.BASE_URL;
```

Comment:
I can't recall exactly why the absolute URL is necessary, but it might have something to do with rendering from a TeamSite subdomain